process-migration/modules/process-migration/module.yaml

```diff
@@ -31,8 +31,8 @@ ports:
 - value: 8080
 artifacts:
 - name: "rhpam_add_ons_distribution.zip"
-  # rhpam-7.12.0-add-ons.zip
-  md5: "8daf93abb6041073a80b4d4be3f1151d"
+  # rhpam-7.12.1-add-ons.zip
+  md5: "130b988a796987f78e5d3f33770e12d9"
```

Comment:
are `RHPAM_PROCESS_MIGRATION_VERSION` and `PRODUCT_VERSION` ok? shouldn't they be `7.12.1`? what about `KIE_PROCESS_MIGRATION_DISTRIBUTION_ZIP`?